### ISSUE TITLE
Ship bundle without dependencies built in

### ISSUE THREAD

**Issue by mAAdhaTTah** [pedronauck/docz#125]
See comment here: https://github.com/pedronauck/docz/issues/118#issuecomment-403176115

I can't test a fix in a dependency because it's is bundled into docz's dist files. Webpack should be able to pull these in on their own.

**Comment by pedronauck** [comment]
I'm bundling `docz` with rollup, `react` and `react-dom` are peerDependencies, so they are not being included in the bundle, but the rest of the dependencies end up being included by the rollup in the bundle.

I do not know, but I'll test putting all dependencies as external to see what happens and if this bug is fixed.

**Comment by pedronauck** [comment]
I released a new version [v0.5.4](https://github.com/pedronauck/docz/releases/tag/v0.5.4) with all dependencies updated. And when I have more time I'll see a permanent solution for that ✌️

**Comment by mAAdhaTTah** [comment]
The downsides are the consumer loses control over what versions are being used (which allows them to lock to versions that work), and the bundler can no longer dedupe if the consumer also uses the same library docz does (e.g. `prop-types`).

**Comment by pedronauck** [comment]
I agree with that, I think that we could found a better solution for this and maybe rethink some architecture decisions. Actually, I think that the biggest pain around docz it's about controlling dependencies (e.g. webpack and babel conflicts) 😕
